c2corg_api/scripts/migration/documents/waypoints/huts.py

```diff
@@ -94,6 +95,16 @@ def get_document_locale(self, document_in, version):
             access_period=document_in.staffed_period
         )
 
+    def set_custodianship(self, document_in):
```

Comment:
You are using the return value of this function to set the attribute. But this function does not set an attribute.